install/README.md

```diff
@@ -8,28 +8,61 @@ To get started with Knative, you need a Kubernetes cluster. If you aren't
 sure which Kubernetes platform is right for you, see
 [Picking the Right Solution](https://kubernetes.io/docs/setup/pick-right-solution/).
 
-We provide information for installing Knative on
-[Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/docs/), [IBM Cloud Kubernetes Service](https://www.ibm.com/cloud/container-service), [Azure Kubernetes Service](https://docs.microsoft.com/en-us/azure/aks/), [Minikube](https://kubernetes.io/docs/setup/minikube/), [OpenShift](https://github.com/openshift/origin) and [Pivotal Container Service](https://pivotal.io/platform/pivotal-container-service) clusters.
+We provide information for installing Knative on multiple different platforms.
 
 ## Installing Knative
 
+There are several options when installing Knative:
+
+* **Bundle install** -- Comes with the default versions of all Knative
+  components. Quickest option for setup.
+
+* **Limited install** -- Installs a subset of Knative components.
```

Comment:
This is not completely accurate because these install packages only reduce or remove what we are now calling "observability plugin" features, and therefore no a "component" the way Serving, Build, and Eventing are.